moneo.py

```diff
@@ -63,20 +97,99 @@ def stop(self):
+
+        print('--------------------------')
+        if self.args.skip_install:
+            pass
+        else:
+            print('-Running install on workers-')
+            logging.info('Running install on workers')
+            pssh(cmd='/tmp/moneo-worker/install/install.sh', hosts_file=hosts_file, max_threads=max_threads)
+            print('--------------------------')
+        print('-Starting metric exporters on workers-')
+        logging.info('Starting metric exporters on workers')
+        cmd = '/tmp/moneo-worker/start.sh'
+        if prof_metrics:
+            cmd = cmd + ' true'
+        else:
+            cmd = cmd + ' false'
+        pssh(cmd=cmd, hosts_file=hosts_file, max_threads=max_threads)
+        print('--------------------------')
+        print('-Deploying Complete')
+
+    def deploy_work_docker(self, hosts_file, max_threads=16):
+        copy_path = './src/worker/deploy_docker.sh'
+        destination_dir = '/tmp/moneo-worker'
+        print('-Deploying docker containers to Nvidia support workers)-')
+        logging.info('Deploying docker container to workers')
+        pscp(copy_path, destination_dir, hosts_file)
+        pssh(cmd='/tmp/moneo-worker/deploy_docker.sh', hosts_file=hosts_file, max_threads=max_threads)
+        print('-Deploying Complete')
+
+    def deploy_manager(self, work_host_file, user=None, manager_host='localhost', export_AzInsight=False):
+        ssh_host = manager_host
+        if user:
+            ssh_host = "{}@{}".format(user, manager_host)
+        copy_path = "src/master/"
+        destination_dir = '/tmp/moneo-master'
+        print('-Copying files to manager-')
+        logging.info('Copying files to manager')
+        cmd = "ssh {} 'rm -rf {}'".format(ssh_host, destination_dir)
+        shell_cmd(cmd, 30)
+        cmd = "scp -r {} {}:{}".format(copy_path, ssh_host, '/tmp/')
+        shell_cmd(cmd, 30)
+        cmd = "ssh {} 'mv {} {}'".format(ssh_host, '/tmp/master', destination_dir)
+        shell_cmd(cmd, 30)
+        print('--------------------------')
+        print('-Deploying Grafana and Prometheus docker containers to manager-')
+        logging.info('Deploying Grafana and Prometheus docker containers to manager')
+        cmd = "ssh {} 'sudo /tmp/moneo-master/managerLaunch.sh {} {}' ".format(ssh_host, work_host_file, manager_host)
+        shell_cmd(cmd, 60)
+        print('--------------------------')
+        if export_AzInsight:
+            print('-Starting Azure insights collector-')
+            logging.info('Starting Azure insights collector')
+            copy_path = "src/azinsights"
+            cmd = "scp -r {} {}:{}".format(copy_path, ssh_host, destination_dir)
+            shell_cmd(cmd, 30)
+            copy_path = "./config.ini"
+            cmd = "scp -r {} {}:{}//azinsights".format(copy_path, ssh_host, destination_dir)
+            shell_cmd(cmd, 30)
+            print('--------------------------')
+        print('-Deploying Complete-')
```

Comment:
Is there proof we made it here in a log somehow? I might be missing some logging.

Reply:
addressed